### DISCUSSION TITLE
Windows

### DISCUSSION THREAD

**Discussion by AfroPK** [TheGlockMisc/eivom-cli#1] [General]
How do you make it work on Windows?

**Comment by Rendevior** [comment]
Widows 10..
You need to turn on WSL(Windows Subsystem for Linux) in "Turn Windows Features on or off" then restart, You need to install ubuntu/debian in "Microsoft Store", you're done setupping it. don't forget to update packages. 
Also needed to install the required Dependencies to run it without errors

Windows 7... i don't think its possible, but you can use VM

**Reply by AfroPK** [reply]
Thanks, mpv on WSL is a bit wonky. Is VLC a viable replacement for WSL?
